pkg/docker/Dockerfile.wasm_wasi_component

```diff
@@ -0,0 +1,108 @@
+FROM debian:bullseye-slim
+
+LABEL org.opencontainers.image.title="Unit (wasm_wasi_component)"
+LABEL org.opencontainers.image.description="Official build of Unit for Docker."
+LABEL org.opencontainers.image.url="https://unit.nginx.org"
+LABEL org.opencontainers.image.source="https://github.com/nginx/unit"
+LABEL org.opencontainers.image.documentation="https://unit.nginx.org/installation/#docker-images"
+LABEL org.opencontainers.image.vendor="NGINX Docker Maintainers <docker-maint@nginx.com>"
+LABEL org.opencontainers.image.version="1.32.0"
```

Comment:
You want to regenerate this file with:

`$ VERSION=1.31.1 make ...`

Reply:
Oh yeah.  That's one of the things that we'd need to fix sometime as we're unable to easily check if Dockefiles can be built with the current code.

I guess this comment can be dropped then.

Reply:
Let's regen with 1.31.1 for internal consistency and just accept that it won't necessarily work off that tag.

Reply:
@thresheek If you're happy for us to merge after regenerating with VERSION=1.31.1, would you mind flipping your review from changes requested to approved? Then Andrew can unilaterally make that one final change and merge.